modules/apps/commerce/commerce-shop-by-diagram-service/src/main/java/com/liferay/commerce/shop/by/diagram/internal/search/spi/model/query/contributor/CSDiagramEntryKeywordQueryContributor.java

```diff
@@ -51,10 +51,18 @@ public void contribute(
 		SearchContext searchContext =
 			keywordQueryContributorHelper.getSearchContext();
 
+		_queryHelper.addSearchLocalizedTerm(
+			booleanQuery, searchContext, Field.DESCRIPTION, false);
 		_queryHelper.addSearchTerm(
 			booleanQuery, searchContext, Field.ENTRY_CLASS_PK, false);
+		_queryHelper.addSearchLocalizedTerm(
+			booleanQuery, searchContext, Field.NAME, false);
+		_queryHelper.addSearchTerm(
+			booleanQuery, searchContext, Field.NAME, false);
 		_queryHelper.addSearchTerm(
 			booleanQuery, searchContext, "sequence", false);
+		_queryHelper.addSearchLocalizedTerm(
+			booleanQuery, searchContext, CPField.SHORT_DESCRIPTION, false);
```

Comment:
@dsitu @ivicac maybe I'm wrong, but I remember that search should be run also in short description, so I believe also SHORT_DESCRIPTION should be added with addSearchTerm to query helper..

Reply:
hmm, good point. I went ahead and added short description to regular search term. Thanks!